react-ui/src/pages/analyzeVotes/AnalyzeVotes.jsx

```diff
@@ -38,10 +38,10 @@ const useStyles = makeStyles({
 });
 
 function UserSelector({
-  user, noVotes, setUser, usernames, title,
+  user, noVotes, setUser, usernames, title, leftRightControlled,
```

Comment:
Where is this prop being set?

Reply:
It was being set by `<UserSelector leftRIghtControlled />`, but I changed it to be a bit cleaner.

Reply:
Ah, I missed that these were two components in the same file. Makes sense.